package.json

```diff
@@ -20,16 +20,12 @@
     "eslint-plugin-react": "^7.13.0",
     "eslint-plugin-react-hooks": "^1.6.0",
     "husky": "^2.3.0",
-    "lint-staged": "^8.1.7"
+    "lint-staged": "^8.1.7",
+    "prettier": "^1.17.1"
```

Comment:
i think you should skipping using pretteir for now, until you guys get familiar with the eslint rules

Reply:
ESLint will be checking for errors and will stop commit,
Prettier will be applied afterward if all goes good.

Prettier is added, So bad codes don't get merged.

Reply:
So with prettier, if you are using vscode and saveOnFormat is on, then it will auto format the file  on saving.

And also if you are checking for eslint errors and fixing them, then whats the use of prettier?

Reply:
ESLint doesn't give errors for extra spaces, 

```js
configure(     { adapter: new Adapter() });
//                ^
//                 |
//                 |
```

Reply:
Ok, then it LGTM

Reply:
It's just for unintentionally committed bad styled code.

If you think that it makes no sense because the team of 3 working on it.
Then it'll be removed.